### PR TITLE
Update local-up-cluster.sh doc to use CRI 

### PR DESCRIPTION
**Pull request by adisky**
**Which issue(s) this PR fixes**:
Update local-up-cluster.sh doc to use CRI

Related to https://github.com/kubernetes/kubernetes/pull/106909

### CODE REVIEW

**Comment by adisky** [comment]
/hold 
(to let merge script update first https://github.com/kubernetes/kubernetes/pull/106909 then will update the doc accordingly)

File: contributors/devel/running-locally.md [hunk 1]
```diff
@@ -27,11 +27,9 @@ Getting started locally
 
 Not running Linux? Consider running [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/), or on a cloud provider like [Google Compute Engine](https://kubernetes.io/docs/setup/production-environment/turnkey/gce/).
 
-### Docker
+### Container Runtime
 
-You will need [Docker](https://docs.docker.com/get-docker/) installed, at least version 17.03. Check the [Kubernetes release notes](https://kubernetes.io/docs/setup/release/notes/) for information about supported Docker versions.
-
-Ensure the Docker daemon is running and can be contacted with `docker ps`. Some of the Kubernetes components need to run as root, which normally works fine with docker.
+You will need [Container Runtime](https://kubernetes.io/docs/setup/production-environment/container-runtimes/) like [Containerd](https://github.com/containerd/containerd) or [CRI-O](https://github.com/cri-o/cri-o) installed and running.
```
Comment (MadhavJivrajani) [inline contributors/devel/running-locally.md:32]:
```suggestion
You will need a [Container Runtime](https://kubernetes.io/docs/setup/production-environment/container-runtimes/) like [Containerd](https://github.com/containerd/containerd) or [CRI-O](https://github.com/cri-o/cri-o) installed and running.
```
nit

**Comment by adisky** [comment]
/ assign @SergeyKanzhelev

**Comment by adisky** [comment]
removing hold since k/k PR is merged
/hold cancel

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

thank you!

**Comment by adisky** [comment]
/assign @nikhita @spiffxp 
(for approval)

**Review by nikhita — APPROVED** [review]
/approve

Thanks!

**Pull request merged**